### ISSUE TITLE
Async functions + spread operator won't compile

### ISSUE THREAD

**Issue by acdlite** [6to5/6to5#620]
After upgrading to 3.0:

```js
function foo(...args) {} // fine
async function foo() // fine
async function foo(...args) {} // won't compile
```

**Comment by dfreeman** [comment]
Also (unsurprisingly) applies to generators.

```javascript
function* foo(...args) {} // won't compile
```

**Comment by sebmck** [comment]
Should be fixed in 3.0.11, I forgot to push out a new release last night when I fixed this issue 2d8944fbd5ee5ee7bb6b6f5a40dd0c4c62385cd0. Regenerator isn't aware of a lot of ES6 nodes so their definitions have to be manually added.
